### ISSUE TITLE
Coverage report contains Control Character symbols 

### ISSUE THREAD

**Issue by igorkotkovets** [openbakery/gradle-xcodePlugin#303]
I running coverage task with parameters (see below) and got control character symbol code 27 ('^[') in the report (screenshot attached). Sonar doesn't accept this report because XML can't contain any control symbols.

```groovy
coverage {
	outputFormat = 'xml'
	exclude = '.h$|.Pods.*m$|.UnitTests.*m$|.ConaxSDKTests.*m$|.NetworkTests.*m$'
}

xcodetest {
    scheme = 'Tests'
    arch = ['x86_64']
    simulator = true
    destination = 'iPad Air 2'
}

xcodebuild {
	scheme = 'MyApp'
	target = 'MyApp'
	configuration = (project.hasProperty('CONFIG') ? CONFIG : 'Release')
	simulator = false
	def mobileProvisionPath = (configuration == 'Debug') ?
			"DefaultBrand/development.mobileprovision" :
			"DefaultBrand/production.mobileprovision"
	def certificatePath = (configuration == 'Debug') ?
			"DefaultBrand/development.p12" :
			"DefaultBrand/production.p12"
	signing {
		mobileProvisionURI = [file(mobileProvisionPath).toURI()]
		certificateURI = file(certificatePath).toURI()
		certificatePassword = ''
	}
}
```
![screen shot 2016-10-17 at 12 56 22 pm](https://cloud.githubusercontent.com/assets/1083951/19433385/23afac68-9469-11e6-8f45-69cfaa1da10d.png)

Could you help?

Thanks!

**Comment by renep** [comment]
Wrong project :-)
The error is in the CoverageReport Tool: https://github.com/openbakery/CoverageReport

**Comment by renep** [comment]
I have updated the CoverageReport tool that hopefully fixes this issue. It takes some time until the jars is deployed in the maven repos and then I want to update the plugin to use the new version.

**Comment by renep** [comment]
should hopefully be fixed with 0.14.3.
If not then reopen this issue with further details.
